packages/amplify-graphql-relational-transformer/src/resolver/rds-generator.ts

```diff
@@ -0,0 +1,234 @@
+import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { ResolverResourceIDs, ResourceConstants } from 'graphql-transformer-common';
+import { MappingTemplate, getPrimaryKeyFields } from '@aws-amplify/graphql-transformer-core';
+import {
+  compoundExpression,
+  ref,
+  set,
+  methodCall,
+  ifElse,
+  printBlock,
+  qref,
+  obj,
+  str,
+  list,
+  Expression,
+  toJson,
+  iff,
+  not,
+} from 'graphql-mapping-template';
+import { BelongsToDirectiveConfiguration, HasManyDirectiveConfiguration, HasOneDirectiveConfiguration } from '../types';
+import { RelationalResolverGenerator } from './generator';
+
+const CONNECTION_STACK = 'ConnectionStack';
+
+export class RDSRelationalResolverGenerator extends RelationalResolverGenerator {
```

Comment:
Most of the code on this file moved from `resolver.ts` to DDB specific resolver generator class.